src/Dependencies/Directory.Build.targets

```diff
@@ -0,0 +1,17 @@
+<Project>
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
+  
+  <!--
+    Include SourcePackage.editorconfig in all source packages.
+  -->
+
+  <PropertyGroup>
+    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);_AddEditorConfigToSourcePackage</TargetsForTfmSpecificContentInPackage>
+  </PropertyGroup>
+
+  <Target Name="_AddEditorConfigToSourcePackage">
+    <ItemGroup>
+      <TfmSpecificPackageFile Include="$(MSBuildThisFileDirectory)SourcePackage.editorconfig" PackagePath="contentFiles/cs/$(TargetFramework)/.editorconfig" />
```

Comment:
I assume you can't just use 'any' for the target framework location, since the actual source files themselves get put under a TFM-specific folder, and that's what the compiler actually is operating on? Put another way, the path the compiler will see is something like:

C:\Users\jasonmal\.nuget\packages\sourcepackage\1.0.0\contentFiles\cs\net8.0\Foo.cs

?

Reply:
That's right.